tests/spread/integration/apache2-utils/task.yaml

```diff
@@ -0,0 +1,10 @@
+summary: Integration tests for apache2
+
+execute: |
+  rootfs="$(apache2_bins dash_bins base-files_bin coreutils_bins adduser_bins)"
```

Comment:
Should this be installing `apache2_utils` instead of `apache2_bins`?

Also, are there any tests we can add for the following?
- /usr/bin/ab
- /usr/bin/checkgid
- /usr/bin/fcgistarter
- /usr/bin/htcacheclean